titles/rosa_cli/images

```diff
@@ -0,0 +1 @@
+/Users/sfortner/dedicated-docs/doc-4/images
```

Comment:
I think this path `sfortner` needs to change.

Reply:
I'm not sure about that yet because we're sharing images. And we haven't published yet in Pantheon 2. I'll check with Sammi and Vikram but I need to close this PR now so I can merge and publish to test that hypothesis.